gitlab/golang/stages/35-management/go.yaml

```diff
@@ -0,0 +1,26 @@
+include:
+  - remote: 'https://raw.githubusercontent.com/rios0rios0/pipelines/main/gitlab/global/abstracts/default-variables.yaml'
+  - remote: 'https://raw.githubusercontent.com/rios0rios0/pipelines/main/gitlab/global/stages/35-management/abstracts.yaml'
+  - remote: 'https://raw.githubusercontent.com/rios0rios0/pipelines/main/gitlab/golang/abstracts/go.yaml'
+
+report:sonarqube:
+  extends: '.sonarqube'
+  script:
+    # Get the latest tag as the current project version
+    - version=$(git describe --tags --abbrev=0)
+    - 'if [ -z "$version" ]; then echo "Could not find the version tag in the repository"; exit 1; fi'
+    # Update sonar-project.properties
+    - echo "sonar.projectVersion=$version" >> sonar-project.properties
+    - echo "Updated sonar.projectVersion to $version"
+    - !reference [ .sonarqube, script ]
+
+report:dependency-track:
+  extends: [ '.go', '.dependency-track' ]
+  script:
+    - go install github.com/CycloneDX/cyclonedx-gomod/cmd/cyclonedx-gomod@latest
+    # Determine the directory to use
+    - dir="cmd"
+    - '[ -d "$(pwd)/main" ] && dir="main"'
+    - cyclonedx-gomod app -json -output "$PREFIX$REPORT_PATH/bom.json" -packages -files -licenses -main $dir
+    - apt update && apt install -y --no-install-recommends curl jq
```

Comment:
Why are dependencies installed here? These are either redundant or they should belong to another stage?

Reply:
Well, it's because each container of each language has its package manager and its initial setup. So, I need to ensure those dependencies are installed before getting back to the default job, which is going to `POST` using `cURL` in the `bom.json` file. Where do you think it is better to use this command?

Reply:
Perhaps consider making an extra job, stage or something to install the dependencies, or just install it in the default job somehow